File: Java/UberBlack.java
package Java;
import java.util.Map;
import java.util.ArrayList;

import Java.Car;
import Java.Driver;

public class UberBlack extends Car{
    Map<String , Map<String,Integer>> typeCardAccepted;
    ArrayList<String> seatsMaterial;

    public UberBlack(String license, 
                     Driver driver, String brand, 
                     String model, Map<String , 
                     Map<String,Integer>> typeCardAccepted,
                     ArrayList<String> seatsMaterial){
        
        super(license, driver);

        this.typeCardAccepted = typeCardAccepted;
        this.seatsMaterial = seatsMaterial;
        

    }
    
}

File: PHP/uberBlack.php
<?php
    require_once('car.php');
    class UberBlack extends Car{
        public $typeCardAccepted;
        public $seatsMaterial;

        public function __construct($typeCardAccepted, 
                                    $seatsMaterial){
            parent::__construct($licence, $driver);

            $this->typeCardAccepted = $typeCardAccepted;
            $this->seatsMaterial = $seatsMaterial;
        }
    }
?>